PendingReleaseNotes

```diff
@@ -25,6 +25,8 @@
    - osd_op_num_shards_hdd = 1 (was 5)
    - osd_op_num_threads_per_shard_hdd = 5 (was 1)
   For more details see https://tracker.ceph.com/issues/66289.
+* RADOS: librados now supports cancellation of async operations via
+  librados::AioCompletion::cancel().
```

Comment:
Classic librados has `rados_aio_cancel()` and `librados::IoCtx::aio_cancel()` which take `AioCompletion` as an argument, so it seems to be exposed already.  Is there a difference in semantics (other than returning void instead of int which is (always?) 0) or is adding a way to do it via a method on `AioCompletion` just needed for ASIO?